src/SearchResults/index.js

```diff
@@ -46,9 +47,19 @@ var generateHierarchicalTree = require('./generate-hierarchical-tree');
 /**
  * @typedef SearchResults.FacetValue
  * @type {object}
- * @property {value} string the facet value itself
- * @property {count} number times this facet appears in the results
- * @property {isRefined} boolean is the facet currently selected
+ * @property {string } value the facet value itself
```

Comment:
Will fix as it breaks the highlighting of github 👍 But it doesn't create issues when rendering the doc.